ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh

```diff
@@ -32,41 +32,8 @@ if [ "${JOB_TYPE}" == "presubmit" ]; then
   EXCLUDE_CNCF_CONFORMANCE=true
 fi
 
-# Implement scenario directory check with fallbacks. Simplify or remove the
-# function when the structure is homogenised in all the active releases.
-function get_source_dir() {
-  local -r base="/home/${HOST_USER}/microshift/test"
-  local -r ndir="${base}/$1"
-  local -r fdir="${base}/$2"
-
-  # We need the variable to expand on the client side
-  # shellcheck disable=SC2029
-  if ssh "${INSTANCE_PREFIX}" "[ -d \"${ndir}\" ]" ; then
-    echo "${ndir}"
-  else
-    echo "${fdir}"
-  fi
-}
-
-if [[ ${JOB_NAME} =~ .*bootc.* ]] ; then
-  if [[ "${JOB_NAME}" =~ .*e2e-aws-tests.*release.* ]]; then
-    SCENARIO_SOURCES=$(get_source_dir "scenarios-bootc/releases" "scenarios-bootc")
-  else
-    SCENARIO_SOURCES=$(get_source_dir "scenarios-bootc/presubmits" "scenarios-bootc")
-    if [[ "${JOB_NAME}" =~ .*periodic.* ]] && [[ ! "${JOB_NAME}" =~ .*nightly-presubmit.* ]]; then
-        SCENARIO_SOURCES=$(get_source_dir "scenarios-bootc/periodics" "scenarios-bootc")
-    fi
-  fi
-else
-  if [[ "${JOB_NAME}" =~ .*e2e-aws-tests.*release.* ]]; then
-    SCENARIO_SOURCES=$(get_source_dir "scenarios/releases" "scenarios")
-  else
-    SCENARIO_SOURCES=$(get_source_dir "scenarios/presubmits" "scenarios")
-    if [[ "${JOB_NAME}" =~ .*periodic.* ]] && [[ ! "${JOB_NAME}" =~ .*nightly-presubmit.* ]]; then
-        SCENARIO_SOURCES=$(get_source_dir "scenarios/periodics" "scenarios-periodics")
-    fi
-  fi
-fi
+SCENARIO_TYPE=${SCENARIO_TYPE:-presubmits}
+SCENARIO_SOURCES=$(get_source_dir "${SCENARIO_TYPE}")
```

Comment:
can we do this in only 1 line? what do you think?
```suggestion
SCENARIO_SOURCES=$(get_source_dir "${SCENARIO_TYPE:-presubmits}")
```

Reply:
Simplified it a bit more, now there is no default.